resources/i18n/en.json

```diff
@@ -1276,7 +1276,8 @@
     "Notifications": "Notifications",
     "NoNotification": "No Notification.",
     "ClearNotifications": "Clear Notifications",
-    "AreYouSureToClearAllNotifications": "Are you sure to clear all notifications?"
+    "AreYouSureToClearAllNotifications": "Are you sure to clear all notifications?",
+    "DestroyAll": "Destroy All"
```

Comment:
I know the API name is `destroy`, How about using `Close` instead of `Destroy` for a user? @agatha197  @ironAiken2

Reply:
<img width="271" alt="image" src="https://github.com/lablup/backend.ai-webui/assets/28584164/ed5883b9-0ed3-412d-be5c-9505235e1724">
How about clear? lol

Reply:
'clear' is fine, but I think it feels more like 'closing' an open popup, rather than clearing space, so 'close' seems more better. How about it?

Reply:
Resolved